names.json

```
@@ -0,0 +1,122 @@
+{
+    "Cors": [
+        "Cross-Origin Resource Sharing",
+        null
+    ],
+    "DAgPbConversion": [
+        "DAG Protobuf Conversion",
+        null
+    ],
+    "DNSLinkGateway": [
+        "DNS Link Gateway",
+        "https://specs.ipfs.tech/http-gateways/dnslink-gateway/"
+    ],
+    "DNSLinkGatewayUnixFSDirectoryListing": [
+        "DNS Link Gateway Unix FS Directory Listing",
+        "https://specs.ipfs.tech/http-gateways/dnslink-gateway/"
+    ],
+    "GatewayBlock": [
+        "Gateway Block",
+        null
+    ],
+    "GatewayCache": [
+        "Gateway Cache",
+        null
+    ],
+    "GatewayCacheWithIPNS": [
+        "Gateway Cache with IPNS",
+        null
+    ],
+    "GatewayIPNSRecord": [
+        "Gateway IPNS Record",
+        null
+    ],
+    "GatewayJSONCborAndIPNS": [
+        "Gateway JSON CBOR and IPNS",
+        null
+    ],
+    "GatewayJsonCbor": [
+        "Gateway JSON CBOR",
+        null
+    ],
+    "GatewaySubdomainAndIPNS": [
+        "Gateway Subdomain and IPNS",
+        null
+    ],
+    "GatewaySubdomains": [
+        "Gateway Subdomains",
+        null
+    ],
+    "GatewaySymlink": [
+        "Gateway Symbolic Link",
+        null
+    ],
+    "NativeDag": [
+        "Native DAG",
+        null
+    ],
+    "Pathing": [
+        "Pathing",
+        null
```

We don't have a dedicated spec for this, but want to write one by EOY.

```suggestion
    "Web Pathing": [
        "Pathing",
        "https://specs.ipfs.tech/http-gateways/path-gateway/#content-resolution"
```